### PR TITLE
UIREQ-936 JEST/RTL test case for ReferredRecord

### PR DESCRIPTION
**Pull request by KetineniM**
UIREQ-936 JEST/RTL test case for ReferredRecord
URL: https://issues.folio.org/browse/UIREQ-936

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

File: src/components/ReferredRecord/ReferredRecord.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  requestCreateDate: '2023-04-05',
+  requesterId: 'testRequesterId',
+  requesterName: 'testRequesterName',
+};
+
+const renderReferredRecord = (prop) => render(
+  <Router history={history}>
+    <ReferredRecord values={prop} />
+  </Router>
+);
```
Comment (artem-blazhko) [inline src/components/ReferredRecord/ReferredRecord.test.js:25]: Why do you need to wrap it into "Router"?

File: src/components/ReferredRecord/ReferredRecord.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import '../../../test/jest/__mock__';
+import React from 'react';
```
Comment (artem-blazhko) [inline src/components/ReferredRecord/ReferredRecord.test.js:2]: Could you remove `import React from 'react';`?

File: src/components/ReferredRecord/ReferredRecord.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  </Router>
+);
+
+describe('ReferredRecord', () => {
+  it('component should render correctly', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.entityType.request')).toBeInTheDocument();
+    expect(screen.getByText('ui-requests.notes.assigned.for')).toBeInTheDocument();
+    expect(screen.getByText('ui-requests.notes.assigned.requester')).toBeInTheDocument();
+    expect(screen.getByText('ui-requests.notes.assigned.requestDate')).toBeInTheDocument();
```
Comment (artem-blazhko) [inline src/components/ReferredRecord/ReferredRecord.test.js:33]: Could you create separate tests for each "expect" statement?

**Review by artem-blazhko — CHANGES_REQUESTED** [review]
Could you add appropriate changes to CHANGELOG.md file?

**Comment by KetineniM** [comment]
> Could you add appropriate changes to CHANGELOG.md file?

Done

File: src/components/ReferredRecord/ReferredRecord.test.js (hunk 2 above)
Comment (KetineniM) [reply src/components/ReferredRecord/ReferredRecord.test.js]: Removed

File: src/components/ReferredRecord/ReferredRecord.test.js (hunk 3 above)
Comment (KetineniM) [reply src/components/ReferredRecord/ReferredRecord.test.js]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,7 @@
 * Cannot save item request from item with status On Order, because need a barcode. Refs UIREQ-924.
 * Cover `src/components/PrintButton/PrintButton.js` file by RTL/Jest tests. Refs UIREQ-886.
 * Cover `src/components/Loading/Loading.js` file by RTL/Jest tests. Refs UIREQ-885.
+* create Jest/RTL test for ReferredRecord.js. Refs UIREQ-936.
```
Comment (artem-blazhko) [inline CHANGELOG.md:22]:
```suggestion
* Create Jest/RTL test for ReferredRecord.js. Refs UIREQ-936.
```

File: src/components/ReferredRecord/ReferredRecord.test.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    expect(screen.getByText('ui-requests.notes.assigned.for')).toBeInTheDocument();
+  });
+  it('assigned.requester should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.assigned.requester')).toBeInTheDocument();
+  });
+  it('assigned.requestDate should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.assigned.requestDate')).toBeInTheDocument();
+  });
```
Comment (artem-blazhko) [inline src/components/ReferredRecord/ReferredRecord.test.js:34]: Could you use `beforeEach` to trigger rendering of component and get rid of rendering inside each test?

Also would be great if you can move all labels like `'ui-requests.notes.assigned.requester'` to separate object `labelIds={}` and then use that object? (see example [here](https://github.com/folio-org/ui-requests/blob/6c369ffea263614f788a0dbe789f48434daf60d7/src/components/ErrorModal/ErrorModal.test.js#L18))

**Review by artem-blazhko — APPROVED** [review]
Looks good!
Thank you!

**Review by Dmitriy-Litvinenko — APPROVED** [review]
Hello. Thank you. Could you please update or rebase branch and merge it?

**Pull request merged**